optima/model.py

```diff
@@ -105,6 +107,8 @@ def model(simpars=None, settings=None, verbose=2, safetymargin=0.8, benchmark=Fa
     circum   = simpars['circum']    # Prevalence of circumcision (P)
     stiprev  = simpars['stiprev']   # Prevalence of STIs (P)
     prep     = simpars['prep']      # Prevalence of PrEP (P)
+    numpmtct = simpars['numpmtct']  # Number (or proportion?) of people receiving PMTCT (P/N)
+    usepmtctprop=True if all(numpmtct<1) else False
```

Comment:
no, what didn't work in v1 was that we checked the entries separaterly on each time loop, like
`if numpmtct[t]>1` for each t.
The broke because it could have been 0 at one time step but a number the next. I think it'd valid to do it this way, however, where you check the whole array at once.